packages/grafana-runtime/src/config.ts

```diff
@@ -160,6 +160,7 @@ export class GrafanaBootConfig implements GrafanaConfig {
     minInterval: '',
     alertStateHistoryBackend: undefined,
     alertStateHistoryPrimary: undefined,
+    recordingRulesEnabled: false,
```

Comment:
Are these not enabled by default?

Reply:
At the time of writing, the `defaults.ini` says no:
https://github.com/grafana/grafana/blob/6f2a9abc0353829869ea9dcbb90a6576c8a41524/conf/defaults.ini#L1530-L1532
But in cloud, I believe it's calculated based on available datasources/something else, and it usually ends up as `true`

Reply:
Resolved offline; let's keep it set to `false`.

> A couple reasons: it requires an external service (not everyone uses Prometheus with Grafana) and we don't have a good UX for enabling without a target (no UI banner, no notice in the API)